example/expression/access_c.py

```diff
@@ -184,5 +185,4 @@ def test(data):
 
 
 if __name__ == '__main__':
```

Comment:
The "__main__" with only one call snippet is not homogeneous with others examples, and only brings an unnecessary complexity (and indent) to the code